CONTRIBUTING.md

```diff
@@ -23,31 +62,42 @@ desired feature or improvement happens is to implement it yourself. The PgJDBC
 sources are reasonably clear and they're pure Java, so it's sometimes easier
 than you might expect.
 
-## Build requirements
+## Contributing code
+
+Here are a few important things you should know about contributing code:
+
+  1. API changes require discussion, use cases, etc. Code comes later.
+  2. Pull requests are great for small fixes for bugs, documentation, etc.
+  3. Pull requests are not merged directly into the master branch.
```

Comment:
Does that mean "pull requests are typically squashed and put as a single commit on top of master branch"?
I would avoid "not" and just phrase the item so it is clear what actually happens.

Reply:
Probably I would rephrase it to something like "Pull request needs to be approved and merged by maintainers into the master branch", others ideas are welcome.

Reply:
That sounds good. I was just puzzled what `not merged directly` meant